java/query/src/test/java/sleeper/query/lambda/SqsQueryProcessorLambdaIT.java

```diff
@@ -640,20 +657,19 @@ public void shouldPublishMultipleStatusReportsToWebSocketForSubQueries() throws
         try {
             // Process Query
             processQuery(query);
-            processQueriesFromQueue(1);
-            processQueriesFromQueue(1);
+            processLeafPartitionQuery(6);
 
             // Then
             wireMockServer.verify(1, postRequestedFor(url).withRequestBody(
```

Comment:
This isn't asserting that these are all the requests that were made. If we add `wireMockServer.verify(5, postRequestedFor(url));` then that would also assert that these are the only requests.

Reply:
Added to test